components/calendar/src/types.rs

```diff
@@ -129,8 +133,8 @@ mod unstable {
         /// ```
         ///
         /// For a full example, see [`Self::ordinal_month`].
-        pub month_code: Option<&'a [u8]>,
-        /// See [`MonthInfo::ordinal`](crate::types::MonthInfo::ordinal).
+        pub month: Option<Month>,
```

Comment:
Issue: This should continue to accept a stringy month code.

Reply:
Seems reasonable